content/posts/irods-development-update-july-2022.md

```diff
@@ -0,0 +1,14 @@
+Title: iRODS Development Update: July 2022
+Date: 2022-07-29 17:00
+Author: Kory Draughn
+Slug: irods-development-update-july-2022
+Status: published
+
+
+July was an exciting month. The [iRODS User Group Meeting](https://irods.org/ugm2022) happened and best of all, it was in person!
+
+It was great seeing everyone and hearing about the interesting things they've been working on using iRODS. We're reviewing footage and hope to post videos very soon.
+
+Our interns are working hard to put the finishing touches on their work. They'll be talking about their experience at [TRiRODS](https://irods.org/trirods) on August 10th. We hope you'll listen in.
```

Comment:
"work" -> "summer work"

mention NFS?  documentation?  s3 glacier deep archive and cleanup?  cmake?  automated ingest character map?

Reply:
Updated.

Let me know if anything needs to be changed.